internal/http/services/owncloud/ocs/conversions/main.go

```diff
@@ -154,6 +154,24 @@ type ShareeData struct {
 	Remotes []*MatchData      `json:"remotes" xml:"remotes>element"`
 }
 
+// TokenInfo holds token information
+type TokenInfo struct {
+	// for all callers
+	Token             string `json:"token" xml:"token"`
+	LinkURL           string `json:"linkURL" xml:"linkurl"`
+	PasswordProtected bool   `json:"passwordProtected" xml:"passwordprotected"`
+
+	// if not password protected
+	StorageID string `json:"storageID" xml:"storageid"`
+	OpaqueID  string `json:"opaqueID" xml:"opaqueid"`
+	Path      string `json:"path" xml:"path"`
+
+	// if native access
+	SpacePath  string `json:"spacePath" xml:"spacepath"`
+	SpaceAlias string `json:"spaceAlias" xml:"spacealias"`
+	SpaceURL   string `json:"spaceURL" xml:"spaceurl"`
```

Comment:
the ocs api uses [snake_case](https://en.wikipedia.org/wiki/Snake_case) check the structs above.

Reply:
Yeah, and the ones below use camelCase :laughing: But ok, then snake_case it is...

Reply:
oh, did I mention that the ocs api has its shortcomings? ;-)